webpack.config.js

```diff
@@ -110,8 +117,17 @@ var config = {
 
 
         ],
+
+        preLoaders: [
+            // All output '.js' files will have any sourcemaps re-processed by 'source-map-loader'.
+            {
+                test: /\.js$/,
+                loader: "source-map-loader"
+            }
+        ]
     },
     'postcss': [require('autoprefixer')],
+    'devtool': 'source-map',
```

Comment:
in latest mast there's an optimization around sourcemap generation.
when we merge, lets make sure we retain it.